packages/sessions-sdk-react/src/token-list.tsx

```diff
@@ -0,0 +1,164 @@
+
+const MotionGridList = motion(GridList<Token>);
+
+type Props = {
+  sessionState: EstablishedSessionState;
+  onPressReceiveTokens: () => void;
+} & (
+  | { onPressToken: (token: Token) => void }
+  | { onPressSend: (token: Token) => void }
+);
+
+export const TokenList = ({
+  sessionState,
+  onPressReceiveTokens,
+  ...props
+}: Props) => {
+  const { faucetUrl, showFaucet } = useFaucet(sessionState);
+  const state = useTokenAccountData(sessionState);
+  switch (state.type) {
+    case TokenDataStateType.Error: {
+      return (
+        <FetchError
+          headline="Failed to fetch your wallet balances"
+          error={state.error}
+          reset={state.reset}
+          className={styles.tokenListError}
+        />
+      );
+    }
+    case TokenDataStateType.Loaded: {
+      return state.data.tokensInWallet.length === 0 ? (
+        <div className={styles.tokenListEmpty}>
+          <WalletIcon className={styles.emptyIcon} />
+          <span className={styles.message}>Your wallet is empty</span>
+          <span className={styles.hints}>
+            <Link onPress={onPressReceiveTokens}>Receive</Link> or{" "}
+            <Link
+              onPress={showFaucet}
+              href={faucetUrl.toString()}
+              target="_blank"
+            >
+              Get tokens
+            </Link>
+          </span>
+        </div>
+      ) : (
+        <MotionGridList
+          layoutId="token-list"
+          className={styles.tokenList ?? ""}
+          selectionMode="none"
+          aria-label="Tokens"
+          items={state.data.tokensInWallet
+            .map((token) => ({ ...token, id: token.mint.toBase58() }))
+            .sort((a, b) => {
+              if (a.name === undefined) {
+                return b.name === undefined
+                  ? a.mint.toString().localeCompare(b.mint.toString())
+                  : 1;
+              } else if (b.name === undefined) {
+                return -1;
+              } else {
+                return a.name.toString().localeCompare(b.name.toString());
+              }
+            })}
+        >
+          {(token) => {
+            const { mint, amountInWallet, decimals, image, name, symbol } =
+              token;
+            const amountAsString = amountToString(amountInWallet, decimals);
+            const contents = (
+              <>
+                <div className={styles.nameAndIcon}>
+                  {image ? (
+                    <img alt="" src={image} className={styles.icon} />
+                  ) : (
+                    <div className={styles.icon} />
+                  )}
+                  <div className={styles.nameAndSymbol}>
+                    <span className={styles.name}>
+                      {name ?? mint.toBase58()}
+                    </span>
+                    {symbol && <span className={styles.symbol}>{symbol}</span>}
+                  </div>
+                </div>
+                <div className={styles.amountAndActions}>
```

Comment:
<img width="349" height="360" alt="Screenshot 2025-09-30 at 1 08 03 PM" src="https://github.com/user-attachments/assets/0c91fe78-2fef-4fc7-af38-286fe06e9e64" />
nit, but i wish i could still see the other tabs when I click onto one of them. like i would like to still see "Receive" rather than having to click "Back"